client/my-sites/post-share/index.jsx

```diff
@@ -339,9 +351,24 @@ class PostShare extends Component {
 
 		return (
 			<div className="post-share__services">
-				<h5 className="post-share__services-header">
-					{ translate( 'Connected services' ) }
-				</h5>
+				<SectionHeader className="post-share__services-header" label={ translate( 'Connected accounts' ) }>
+					<Button
+						compact
+						href={ '/sharing/' + siteId }
+						className="post-share__add-button"
+						onMouseEnter={ this.showAddTooltip }
+						onMouseLeave={ this.hideAddTooltip }
+						ref="addAccountButton"
+						aria-label={ translate( 'Add account', { context: 'button label' } ) }>
```

Comment:
Not necessary in this case, we already have it translated without the context. It's always ok to ask :)
My rule for adding a context: only add it if I know it can have more than one interpretation (i.e. "Post" being a verb or a noun), or if I'm reusing an existing string but with a different meaning.

It's easy to add a context later, it's harder to remove it when it's been added.